prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -307,23 +316,16 @@ struct fi_ibv_rdm_tagged_postponed_entry {
 	struct fi_ibv_rdm_tagged_conn *conn;
 };
 
-enum fi_ibv_rdm_buffer_status {
-	BUF_STATUS_FREE = 0,
-	BUF_STATUS_BUSY
-};
-
+/*
+ * Size of the structure is performance sensitive. TODO: resize to 4 bytes.
+ */
 struct fi_ibv_rdm_buffer_service_data {
 	volatile enum fi_ibv_rdm_buffer_status status;
```

Comment:
Sounds reasonable but in this case ep_rdm has more issues regarding endian format that I see because currently everything is sending with rdma write op code. CM, eager and rndv protocols also should be modified. In addition, which level should control user's data including rma and atomics? I'm guessing user?